llvm/test/CodeGen/RISCV/switch-width.ll

```diff
@@ -194,25 +194,108 @@ return:
   %retval = phi i32 [ -1, %sw.default ], [ 0, %sw.bb0 ], [ 1, %sw.bb1 ]
   ret i32 %retval
 }
+define i32 @sext_i32(i16 %a)  {
```

Comment:
You should add test variants here with zeroext and signext on the arguments to make sure we test the interaction.